src/FixerDefinition/VersionSpecification.php

```diff
@@ -36,19 +36,21 @@
     public function __construct($minimum = null, $maximum = null)
     {
         if (null === $minimum && null === $maximum) {
-            throw new \InvalidArgumentException('Either minimum or maximum need to be specified');
+            throw new \InvalidArgumentException('Minimum and/or maximum need to be specified.');
```

Comment:
>Either minimum or maximum

means either one of them, so not both?

>Minimum or maximum 

mean one of the two but not both?

I dunno, maybe bin the PR?